arian1917/arian1917/Program.cs

```diff
@@ -0,0 +1,579 @@
+﻿using System.Data;
+using System.Diagnostics.Metrics;
+using System.Globalization;
+using System.Linq.Expressions;
+using System.Numerics;
+using System.Reflection.Metadata.Ecma335;
+using System.Runtime.CompilerServices;
+using Microsoft.Data.Sqlite; 
+
+
```

Comment:
Whitespace: Try not to include double line breaks. It unnecessarily inflates the line count of your files.